primefaces/src/main/java/org/primefaces/component/splitbutton/SplitButtonRenderer.java

```diff
@@ -201,7 +201,7 @@ protected void encodeMenuIcon(FacesContext context, SplitButton button, String i
 
         //icon
         writer.startElement("span", null);
-        writer.writeAttribute("class", "ui-button-icon-left ui-icon ui-icon-triangle-1-s", null);
+        writer.writeAttribute("class", HTML.BUTTON_LEFT_ICON_CLASS + " ui-icon-triangle-1-s", null);
```

Comment:
Do we still need `ui-icon-triangle-1-s`?

Reply:
Yep that is is the dropdown icon itself.  It was missing `ui-c` but instead of adding `ui-c` there is already a whole LEFT_ICON_CLASS just for putting the icon correctly, but naming which icon is still important.

Reply:
That is still missing `ui-c` right?  Which was the whole genesis of the ticket as Datatale ignores all things with `ui-c` in it?

Reply:
Yes, that's unrelated to the `ui-c` thing. Maybe this should refactored in 14 when custom icons are introduced.

Reply:
Yeah I think probably all the icons need to be revisted.